using-timescaledb/continuous-aggregates.md

```diff
@@ -22,7 +22,11 @@ automatic, it doesn’t add any maintenance burden to your database.
 raw data from the hypertable and computes a partial aggregate that it
 stores (materializes) in the continuous aggregate.
 
-Querying the *continuous aggregate view* will then compute an
+
+**Real-Time Aggregate** 
```

Comment:
```suggestion
**Real-Time Aggregates** 
```

Reply:
I dont know maybe its too late for that but i really like to call it Real-Time Aggregation cause that works much better in conjunction with continuous aggregates